blaze/expr/collections.py

```diff
@@ -424,10 +424,36 @@ def join(lhs, rhs, on_left=None, on_right=None, how='inner'):
 
 join.__doc__ = Join.__doc__
 
+class IsIn(Expr):
+    """
+    """
+    __slots__ = '_hash', '_child', '_key'
+
+    def __init__(self, child, key):
+        super(IsIn, self).__init__(child, frozenset(key))
```

Comment:
sorry for the churn, your original idea of putting this in the `isin` function is better :)